src/pages/generic/index.js

```diff
@@ -1,12 +1,14 @@
 import supportedEvents from '../../components/functions/supported.events.vue';
 import supportedFunctions from '../../components/functions/supported.functions.vue';
 import supportedRequests from '../../components/functions/supported.requests.vue';
+import rpcMetadata from '../../components/functions/supported.metadata.vue';
```

Comment:
Move the updated docs from components -> pages like the ones that have been updated already so it's easier to track what has been migrated to the new structure.

Reply:
Check.